packages/integration-tests/src/components/accessibility/test-delegates-focus-focus-method-on-host-element/delegates-focus-focus-method-on-host-element.spec.js

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
This is the main content of this PR (everything else is just moving tests around). Invoking the focus method of a component that is delegating focus should place focus on the host element. It is up to the component to decide where the focus should go by implementing its own focus method.